src/hooks/useUserLocation.js

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react'
+import getGeoData from '../services/getGeoData'
+import getRkiData from '../services/getRkiData'
+
+export default function useUserLocation() {
+
+    const [userPlace, setUserPlace] = useState('')
```

Comment:
"Place" sounds weird somehow, "location" might be misleading... maybe something that has to do with what the user actually inputs? Not sure, maybe location is actually not bad because it could be a city or street or zipcode